trade-finance-manager-ui/templates/case/activity/activity-comment.njk

```diff
@@ -5,9 +5,9 @@
 
 {% extends "index.njk" %}
 
-{% block pageTitle %}
+{% block pageTitle -%}
```

Comment:
Can all the following instances be replaced with following?

```nunjucks
{% block pageTitle %}
  - Title - 
{% endblock %}
```

Reply:
I updated all pageTitle blocks